backend/routes/FormRoutes.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const formController = require('../controllers/formController.js')
```

Comment:
Extract the methods directly from formController
```suggestion
const {getFormsByEventId}= require('../controllers/formController.js')
```